tests/test_client.py

```diff
@@ -147,3 +153,225 @@ def on_message(client, userdata, msg):
 
         packet_in = fake_broker.receive_packet(1)
         assert not packet_in  # Check connection is closed
+
+
+@pytest.mark.parametrize("proto_ver,proto_name", [
+    (client.MQTTv31, "MQIsdp"),
+    (client.MQTTv311, "MQTT"),
+])
+class TestInvalidWebsocketResponse(object):
```

Comment:
Makes sense, I just put them in there because the file already existed. I'll move it to test_websockets.py or something similar